### PR TITLE
Amit mandhane canton

### LINKED ISSUE #14: Amit's todo list
- [x] Create an issue
- [ ] Create a branch

### PR DESCRIPTION
**Pull request by AmitMandhane**
Adding new files.

Closes #14

### CODE REVIEW

File: NewFile.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+##
+###
+####
+#####
+######
+
+
+- Sponge Docks
+- Dunedin Brewery :beers:
+- Casa Tina
```
Comment (devops23345) [inline NewFile.md:11]: Is the salsa spicy?

**Comment by githubteacher** [comment]
Do you have any other restaurant suggestions?

File: NewFile.md (hunk 1 above)
Comment (AmitMandhane) [reply NewFile.md:11]: Yes @nzxhvr

**Comment by AmitMandhane** [comment]
New restaurant added

**Pull request merged**